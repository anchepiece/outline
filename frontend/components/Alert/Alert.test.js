/* eslint-disable */
import React from 'react';
import Alert from '.';

test('renders default as info', () => {
  snap(<Alert>default</Alert>);
});

test('renders success', () => {
  snap(<Alert success>success</Alert>);
});

test('renders info', () => {
  snap(<Alert info>info</Alert>);
});

test('renders warning', () => {
  snap(<Alert warning>warning</Alert>);
});

test('renders danger', () => {
  snap(<Alert danger>danger</Alert>);
});
