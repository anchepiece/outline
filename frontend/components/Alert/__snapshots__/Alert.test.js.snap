// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders danger 1`] = `
<Flex
  align="center"
  className="container danger"
  justify="center"
>
  danger
</Flex>
`;

exports[`renders default as info 1`] = `
<Flex
  align="center"
  className="container info"
  justify="center"
>
  default
</Flex>
`;

exports[`renders info 1`] = `
<Flex
  align="center"
  className="container info"
  justify="center"
>
  info
</Flex>
`;

exports[`renders success 1`] = `
<Flex
  align="center"
  className="container success"
  justify="center"
>
  success
</Flex>
`;

exports[`renders warning 1`] = `
<Flex
  align="center"
  className="container warning"
  justify="center"
>
  warning
</Flex>
`;
